cluster/service/cluster_service_blackbox_test.go

```diff
@@ -696,10 +726,9 @@ func (s *ClusterServiceTestSuite) TestList() {
 					result, err := s.Application.ClusterService().List(ctx, nil)
 					// then
 					require.NoError(t, err)
-					require.Len(t, result, 3)
 					expected, err := repository.NewClusterRepository(s.DB).List(ctx, nil)
 					require.NoError(t, err)
-					test.AssertEqualClusters(t, expected, result, false)
+					test.AssertContainsClusters(t, expected, result, false)
```

Comment:
I'm not sure why these assertions were changed - technically, it should be still valid - that the result should contain the clusters stored in the DB. I reverted it locally and the unit tests are still green.

Reply:
These are integration tests. Not unit. It looks like the DB is not cleared properly in tests which cause some unexpected noise in the tests. The tests were failing for me even in master without that change. Maybe there are some race conditions in the tests...

Reply:
ok, thanks for explanation :+1: